src/main/resources/application.yml

```diff
@@ -45,6 +49,30 @@ spring:
       continue-on-error: true
   jackson:
     default-property-inclusion: non_null
+  quartz:
+    auto-startup: ${folio.quartz.enabled}
+    job-store-type: jdbc
+    jdbc:
+      initialize-schema: never
+    properties:
+      org.quartz:
+        scheduler:
+          instanceId: AUTO
+        jobStore:
+          driverDelegateClass: org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+          class: org.quartz.impl.jdbcjobstore.JobStoreTX
+          tablePrefix: ${folio.quartz.schemaName}.QRTZ_
+          dataSource: quartz
+          misfireThreshold: 10000
+          isClustered: true
+          clusterCheckinInterval: 15000
+        dataSource:
+          quartz:
```

Comment:
By default quartz is working with a default spring datasource, however in folio modules this bean is replaced by [DataSourceFolioWrapper](https://github.com/folio-org/folio-spring-base/blob/6c98ec7af23963b59ae7d9c114432484834bd5fc/src/main/java/org/folio/spring/config/DataSourceFolioWrapper.java) which chooses schema based on the tenant from FolioExecutionContext. It's not needed for quartz (its tables are located in a separate, not tenant-specific schema) + it produces warn log entries about context not set.
 So defined another datasource with  same configs as default.

Reply:
Created SchedulerFactoryBeanCustomizer to use plain datasource for quartz instead of DataSourceFolioWrapper. Removed separate quartz datasouce declaration from properties.